domains/lowsummer.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Site Low Summer",
+    "repo": "https://github.com/LowSummer/lowsummer.github.io",
+    "owner": {
+        "username": "LowSummer",
+        "email": "<email@address>",
+        "twitter": "<twitter-username>"
```

Comment:
Socials are invalid, provide ones that have valid entries. You can use owl, email, or another valid way of contact.

Reply:
I will correct it